src/main/java/de/azapps/kafkabackup/sink/BackupSinkTask.java

```diff
@@ -206,6 +206,7 @@ public void stop() {
             }
             offsetSink.close();
             log.info("Stopped BackupSinkTask");
+            exitFunction.accept(0);
```

Comment:
We shoud not simply stop Kafka Connect when we stop this task. I think this should be done using a new flag similarly to #99: 
https://github.com/itadventurer/kafka-backup/pull/99/files#diff-ba5c35b2ef715943238042d755657cb4R125

There is also an open issue for that: #101. Could you please split this PR in two?

Reply:
Alright, makes sense to have a flag for it.